### ISSUE TITLE
lr =0 when freezing the backbone and neck for transfer learning

### ISSUE THREAD

**Issue by yue-cheng-wind** [open-mmlab/mmocr#958]
Hi:
I would like to train my text detection model on my custom datasets, using the transfer learning technique. When I launched the training, I used 'load_from' to specify the pretrained model downloaded from the mmocr page. The training results worked good. 

Then, I would like to freeze the  backbone and neck referring to the [architecture](https://mmocr.readthedocs.io/en/latest/model_summary.html) and this [post](https://github.com/open-mmlab/mmocr/issues/828)

I thus modify the optimizer config, as the post suggested above, and found out that the lr in the training log became close to zero, and the loss converges much slower than that without freezing the backbone and the neck.

Below is my log file:
[20220421_012433 (2).log](https://github.com/open-mmlab/mmocr/files/8526593/20220421_012433.2.log)

Could you please suggest if there is any part that I did it wrong? Thank you so much for your help in advance!

**Comment by gaotongxiao** [comment]
The loss is still decreasing, so it's actually just a bug in computing the overall learning rate. We'll get it fixed asap. Thanks for reporting that!

**Comment by yue-cheng-wind** [comment]
@gaotongxiao  Hi Tong! Thank you so much for your prompt reply! Will wait for your updates!
